tools/deployment/make_linux_package.sh

```diff
@@ -125,7 +126,12 @@ function main() {
     cp $OSQUERY_CONFIG_SRC $INSTALL_PREFIX/$OSQUERY_ETC_DIR/osquery.conf
   fi
 
-  if [[ $DISTRO = "centos7" || $DISTRO = "rhel7" || $DISTRO = "xenial" ]]; then
+  if [[ $DISTRO = "xenial" ]]; then
+  #Change config dir path for Xenial
+  sed -i 's/sysconfig/default/g' $SYSTEMD_SERVICE_SRC
```

Comment:
Why are you changing the `SRC` here?

```bash
SYSTEMD_SERVICE_SRC="$SCRIPT_DIR/osqueryd.service"
```

Would become:
```bash
SYSTEMD_SERVICE_SRC="$SCRIPT_DIR/osqueryd.service"
```

No change! I think you want `SYSTEMD_SYSCONFIG_DST`